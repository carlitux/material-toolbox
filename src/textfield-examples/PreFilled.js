// @flow
import * as React from 'react';

import Example from '../Example';
import TextField from '../material-toolbox/textfield/TextField';

const source = `import TextField from 'material-toolbox/textfield/TextField';

const PreFilled = () => (
  <TextField defaultValue="Pre-filled value" label="Label floating above" />
);

export default PreFilled;
`;

const PreFilled = () => (
  <Example title="Preventing FOUC" source={source}>
    <TextField defaultValue="Pre-filled value" label="Label floating above" />
  </Example>
);

export default PreFilled;
