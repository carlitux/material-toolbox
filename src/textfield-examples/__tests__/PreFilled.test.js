// @flow
import React from 'react';
import { shallow } from 'enzyme';
import PreFilled from '../PreFilled';

describe('example::PreFilled', () => {
  it('Should match snapshoot', () => {
    const component = shallow(<PreFilled />);
    expect(component).toMatchSnapshot();
  });
});
