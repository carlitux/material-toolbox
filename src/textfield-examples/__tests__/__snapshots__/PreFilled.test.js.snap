// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example::PreFilled Should match snapshoot 1`] = `
<Example
  source="import TextField from 'material-toolbox/textfield/TextField';

const PreFilled = () => (
  <TextField defaultValue=\\"Pre-filled value\\" label=\\"Label floating above\\" />
);

export default PreFilled;
"
  title="Preventing FOUC"
>
  <TextField
    box={false}
    cssOnly={false}
    defaultValue="Pre-filled value"
    dense={false}
    disabled={false}
    fullWidth={false}
    invalid={false}
    label="Label floating above"
    multiline={false}
    outlined={false}
    persistentHelperText={false}
    validationHelperText={false}
  />
</Example>
`;
